Dev video 16 9

Briggs-Richardson opened the pull request:
DO NOT MERGE

Briggs-Richardson committed:
Sets the aspect ratio of the video widget to 16:9 with window sizing assumptions

Briggs-Richardson committed:
Fix temporary testing code left in previous commit. The table should now have a minimum height for 5 rows again

Briggs-Richardson committed:
Sets a preferred 16:9 AR to the video widget

This patch sets a preferred 16:9 aspect ratio on the video widget. This
patch also establishes a minimum size on the video widget.

If the widget has the right amount of vertical space for its width, then
it will adjust to a 16:9 aspect ratio. However, if it does not, then
black borders will be added to the video to maintain a 16:9 video
display aspect ratio.

In the future, a different strategy may be implemented in order to
guarantee a 16:9 aspect ratio regardless of the aspect ratio of the main
window.

Testing Steps
  1. Run main.py to start the application.
  2. Load and run a local video file that has a 16:9 aspect ratio.
  3. Verify that there exists no black bars on the video widget when the
     main application window has a 16:9 aspect ratio.
  4. Decrease the width of the main window and verify that the video
     widget maintains a 16:9 aspect ratio.
  5. Verify that the minimum size of the video widget is 500px x 281px.

Type: New Feature

Jcobo314 commented on View/video_widget.py:
```diff
@@ -17,9 +17,11 @@ def __init__(self):
+        self.setMinimumHeight(self.heightForWidth(self.minimum_width))
 
     def heightForWidth(self, width):
         """The video widget height should be 9/16 of the width."""
```
Comments should be 
"""
*comment*
"""

Briggs-Richardson committed:
Sets a preferred 16:9 AR to the video widget

This patch sets a preferred 16:9 aspect ratio on the video widget. This
patch also establishes a minimum size on the video widget.

If the widget has the right amount of vertical space for its width, then
it will adjust to a 16:9 aspect ratio. However, if it does not, then
black borders will be added to the video to maintain a 16:9 video
display aspect ratio.

In the future, a different strategy may be implemented in order to
guarantee a 16:9 aspect ratio regardless of the aspect ratio of the main
window.

Testing Steps
  1. Run main.py to start the application.
  2. Load and run a local video file that has a 16:9 aspect ratio.
  3. Verify that there exists no black bars on the video widget when the
     main application window has a 16:9 aspect ratio.
  4. Decrease the width of the main window and verify that the video
     widget maintains a 16:9 aspect ratio.
  5. Verify that the minimum size of the video widget is 500px x 281px.

Type: New Feature
